starburst-presto/presto.sh

```diff
@@ -123,23 +128,35 @@ function configure_hive(){
 connector.name=hive-hadoop2
 hive.metastore.uri=${metastore_uri}
 EOF
+
+# Add new connectors configs here 
+# For example: Adding jmx properties like
+  cat > presto-server-${PRESTO_VERSION}/etc/catalog/tpch.properties <<- "EOF"
+connector.name=tpch
+tpch.splits-per-node=4
+EOF
+
```

Comment:
Great idea. As an optional suggestion, could you add the tpcds connector as well?

Reply:
One will enough right? I added that so users can understand how users can connectors.

If you want I can add jmx and tpcds as well.

Reply:
one is fine. It was only a suggestion since TPCH and TPCDS seem natural to go together and didn't seem like a ton of work to add both.